libs/ledger-live-common/src/apps/support.ts

```diff
@@ -33,7 +33,7 @@ const appVersionsRequired = {
   Algorand: ">= 1.2.9",
   Polkadot: ">= 13.9250.0",
   Elrond: ">= 1.0.11",
-  Ethereum: ">= 1.9.17",
+  Ethereum: ">= 1.9.20",
```

Comment:
do you confirm this version is available on latest firmware of all devices? (nano s, nano x, ..)

if so, it's good.

Reply:
We will merge when the version is ready, for now it's now available I guess